bpf/bpf_lxc.c

```diff
@@ -742,12 +742,12 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 
 	orig_dip = ip4->daddr;
 
-	/* Allow a hairpin packet to be redirected even if ENABLE_ROUTING is
+	/* Allow a hairpin packet to be redirected even if ENABLE_ENDPOINT_ROUTES is
 	 * disabled. Otherwise, the packet will be dropped by the kernel if
```

Comment:
Comment does not look correct since the value has changed.

Reply:
will correct the comment